src/URI.js

```diff
@@ -366,6 +402,22 @@
 
     return segments.join('/');
   };
+  URI.recodeURNPath = function(string) {
```

Comment:
`URI.recodeUrnPath`

this method looks identical to `recodePath()` except for the delimiter and the recode callback. this should probably be refactored to a generator like `generatePrefixAccessor()` does for accessors